pkg/status.go

```diff
@@ -174,14 +174,14 @@ func ChaosPodStatus(testsDetails *types.TestDetails, clients environment.ClientS
 		if err != nil {
 			return errors.Errorf("fail to get the chaosengine %v err: %v", testsDetails.EngineName, err)
 		}
-		if len(chaosEngine.Status.Experiments) == 0 {
+		if chaosEngine.Status.Experiments) == "" {
```

Comment:
The type for the `chaosEngine.Status.Experiments` is `[]v1alpha1.ExperimentStatuses` which is a list only so we can retain the older change.